api/v4/source/introduction.yaml

```diff
@@ -49,6 +49,9 @@ tags:
 
 
       When using endpoints that require a user id, the string `me` can be used in place of the user id to indicate the action is to be taken for the logged in user.
+
+
+      For all endpoints that implement pagination via the `per_page` parameter, the maximum number of items returned per request is capped at 200.
```

Comment:
It would be also good to clarify that if a number more than 200 is passed, it's silently truncated to 200.

Reply:
Done